release/ray_release/buildkite/step.py

```diff
@@ -121,13 +121,16 @@ def get_step(
     if test.get("run", {}).get("type") == "client":
         step["agents"]["queue"] = str(RELEASE_QUEUE_CLIENT)
 
-    # If a test is not stable, allow to soft fail
+    # If a test is not stable or jailed, allow to soft fail
     stable = test.get("stable", True)
-    if not stable:
+    jailed = test.get("jailed", True)
+    full_label = ""
+    if not stable or jailed:
         step["soft_fail"] = True
```

Comment:
nit: I feel like this can be a bit confusing, may be helpful to reverse the order to make it explicit that the "not" does not apply to jailed:

```python
# If a test is jailed or not stable, allow to soft fail
...
if jailed or not stable:
```

Reply:
love it!